src/data_table/local_data_table.js

```diff
@@ -77,7 +82,7 @@ class LocalDataTable extends View {
     this._enableRowHighlight();
     this.paginate && this._initPaginationHandling();
     this._triggerChangeSelection();
-    this.paginate && this._setupPaginationHistory();
+    this.urlPagination && this._setupPaginationHistory();
```

Comment:
This should check both flags right?  Or, the above lines should ensure that urlPagination is ONLY set true if `paginate` is also true.. if the above does that, code below can be simpler probably.. otherwise you will need to always be sure to check both

Reply:
Looks like you just pushed a commit for this.  I think still better to protect against these flags drifting though

Reply:
I just addressed that the moment before you made this comment 😂 , but yes it did seem inconsistent that I would drop the paginate check.